frontend/src/metabase/actions/containers/ActionCreator/tests/ActionCreator-ImplicitActions.unit.spec.ts

```diff
@@ -36,7 +36,7 @@ describe("ActionCreator > Implicit Actions", () => {
       }),
     });
 
-    expect(screen.getByText("FooBar")).toBeInTheDocument();
+    expect(screen.getAllByText("FooBar")).toHaveLength(2);
```

Comment:
Field title is now rendered twice:

![CleanShot 2023-03-03 at 17 05 22@2x](https://user-images.githubusercontent.com/17258145/222782740-61a2e0a0-4727-48a8-897e-fdae36ba3351.png)